extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/BridgeHandler.java

```diff
@@ -199,8 +200,12 @@ private Config loadAndCheckConfig() {
         if (StringUtils
                 .isNotBlank(thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL).toString())) {
             try {
-                config.setSensordataRefreshInterval(Integer.parseInt(
-                        thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL).toString() + "000"));
+                Object param;
+                param = thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL);
+                logger.debug("param = {}", param);
+                if (param instanceof BigDecimal && param != null) {
+                    config.setSensordataRefreshInterval(((BigDecimal) param).intValue());
+                }
             } catch (NumberFormatException e) {
```

Comment:
The catch block is not reachable now.  Use an else-block at the `if( ... instanceof BigDecimal){ ...` for the error message e.g.

if (thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL) instanceof BigDecimal) {
        config.setSensordataRefreshInterval(((BigDecimal)   thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL)).intValue()  * 1000);
        } else {
            if (numberExc == null) {
                numberExc = new ArrayList<String>();
            }
            numberExc.add("\"Sensor update interval\" ( "
                    + thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL) + ")");
        }

Also at the next try-catch-blocks.